### ISSUE TITLE
Error when planning against an imported existing resource: insufficient items for attribute "domain"

### ISSUE THREAD

**Issue by goodspark** [fastly/terraform-provider-fastly#183]
### Terraform Version
```
Terraform v0.12.6
+ provider.fastly v0.8.1
```

### Affected Resource(s)
- fastly_service_v1

### Terraform Configuration Files
```hcl
provider "fastly" {
  # I have the API key in my env vars
}

resource "fastly_service_v1" "spark_test" {
  name = "spark-test.xxxx.com"

  domain {
    name    = "spark-test.xxxx.com"
    comment = "spark testing"
  }
}
```

### Debug Output
https://gist.github.com/goodspark/d24547bfc067823c5d63f71cf53664a9

### Expected Behavior
A plan output with no changes.

### Actual Behavior
Error message: `insufficient items for attribute "domain"; must have at least 1`

### Steps to Reproduce
1. Create a new service manually in Fastly
1. Add a single domain and nothing else
1. Import this service into your local terraform state for the above HCL
1. Run 'terraform plan'

### Important Factoids
- Here's a screenshot of my Fastly service config. It's very minimal - I just wanted to first verify I could import an existing service.
![Screenshot from 2019-08-01 16-34-11](https://user-images.githubusercontent.com/29210237/62334089-0e92f580-b47b-11e9-93bc-a1f57f5424fe.png)

- If I remove the existing service from my local state and run plan, it's successful.
- But what I want to do here is import an existing service and start managing it in Terraform

**Comment by jlav1n** [comment]
For the record, I do not see this with:
```
Terraform v0.11.7
+ provider.fastly v0.8.1
```

**Comment by ikbenale** [comment]
This is possibly related to https://github.com/terraform-providers/terraform-provider-aws/issues/9048 and therefore possibly addressed by https://github.com/hashicorp/terraform/pull/22478

**Comment by jlav1n** [comment]
Hmm, using Terraform v.0.12.7 I have the same issue as the original poster. (edit: also with 0.12.9 -- but yeah, the Fastly provider would need to vendor, as currently on 0.12.3)

**Comment by ikbenale** [comment]
I think hashicorp/terraform#22478 will go into v0.12.8 but either way, terraform-provider-fastly may have to vendor?

**Comment by marcpalm** [comment]
The requirements state Terraform 0.10.x. Is it working with 0.12.*?

**Comment by jlav1n** [comment]
This is fixed for me using Fastly provider version `0.11.0`

**Comment by smaeda-ks** [comment]
It looks like this was resolved a while ago. I'm going to close this issue.
